hooks/git_commit_standards_hook.py

```
@@ -34,6 +34,38 @@ def main():
+                        "❌ NEVER commit directly to main/master! Per CLAUDE.md GitHub Flow:\n\n"
+                        "1. Create feature branch:\n"
+                        "   git checkout -b feature/your-feature\n\n"
+                        "2. Make changes and commit to feature branch\n\n"
+                        "3. Push feature branch:\n"
+                        "   git push origin feature/your-feature\n\n"
+                        "4. Create Pull Request for review\n\n"
+                        "This ensures code review and prevents breaking production."
+                    )
+                }))
+                return 1
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)